src/quo2/components/inputs/search_input/view.cljs

```diff
@@ -0,0 +1,88 @@
+(ns quo2.components.inputs.search-input.view
+  (:require [oops.core :as oops]
+            [quo2.components.icon :as icon]
+            [quo2.components.inputs.search-input.style :as style]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]))
+
+(def ^:private tag-separator [rn/view {:style style/tag-separator}])
+
+(defn- inner-tags
+  [tags-coll]
+  (into [rn/view {:style style/tag-container}]
+        (interpose tag-separator)
+        tags-coll))
+
+(defn- clear-button
+  [{:keys [on-press blur? override-theme]}]
+  [rn/touchable-opacity
+   {:style    style/clear-icon-container
+    :on-press on-press}
+   [icon/icon :i/clear
+    {:color (style/clear-icon blur? override-theme)
+     :size  20}]])
+
+(defn handle-backspace
+  [^js/Object event scroll-view-ref]
+  (when (= (oops/oget event "nativeEvent.key") "Backspace")
```

Comment:
just wondering if there are constants for keys

Reply:
Not yet, do you find it useful? I could create it if needed.

ATM, I think we can just use strings until they get more common